module/zfs/vdev_queue.c

```diff
@@ -313,10 +313,10 @@ vdev_queue_class_min_active(vdev_queue_t *vq, zio_priority_t p)
 	}
 }
 
-static int
+static uint_t
 vdev_queue_max_async_writes(spa_t *spa)
 {
-	int writes;
+	uint32_t writes;
```

Comment:
Why 32 if function returns uint_t and tunables assigned to it are uint_t?

Reply:
Fixed